.github/workflows/ci-llama.yaml

```diff
@@ -9,6 +9,10 @@ name: Llama Benchmarking Tests
 on:
   workflow_dispatch:
   pull_request:
+    paths:
+      - '.github/workflows/ci-llama.yaml'
+      - 'sharktank/**'
+      - '*requirements*.txt'
```

Comment:
I think for the moment we should actually remove all these path filters from the workflows and start switching more checks to be required. Too many developers are ignoring CI results and relying on auto-merge, wasting multiple developer-hours a week on reverts and fix-forwards.

We could build a system like https://iree.dev/developers/general/github-actions/#opt-in-for-presubmit-jobs to get the best of both worlds (filtering _and_ path filters), since GitHub's default implementation is completely broken :|

Reply:
I generally agree, but don't have a good feeling yet which workflows would be a good choice to be run on every PR. However, I clearly see the point. Closing this therefore.

Reply:
Every workflow that runs in less than 30 minutes.